### PR TITLE
Add Apache logs to GreyNoise Schemas, add Context theme

### PR DESCRIPTION
**Pull request by bseb**
### Background
Add Apache log sources to GreyNoise Lookup Tables
Add Context function to GreyNoise Helpers
Fix bug with generating a URL in the RIOT Classes 

### Changes
See Above

### Testing

Unit Tests, Manual testing in Internal Security Panther

### CODE REVIEW

File: lookup_tables/greynoise/basic/riot_basic.yml [hunk 1]
```diff
@@ -14,6 +14,12 @@ LogTypeMap:
     - LogType: AlphaSOC.Alert
       Selectors:
         - "$.event.srcIP"
+    - LogType: Apache.AccessCombined
```
Comment (bseb) [inline lookup_tables/greynoise/basic/riot_basic.yml:17]: @rleighton obviously not a complex fix but I wanted to get your 👍🏼  on the LUT and Pack yaml updates

File: global_helpers/panther_greynoise_helpers.py [hunk 2]
```diff
@@ -54,6 +54,13 @@ def actor(self, match_field) -> str:
     def url(self, match_field) -> str:
         return f"https://www.greynoise.io/viz/ip/{deep_get(self.noise, match_field, 'ip')}"
 
+    def context(self, match_field) -> dict:
```
Comment (lindsey-w) [inline global_helpers/panther_greynoise_helpers.py:57]: handy!

File: global_helpers/panther_greynoise_helpers.py [hunk 3]
```diff
@@ -54,6 +54,13 @@ def actor(self, match_field) -> str:
...
+        return{
+            "IP": self.ip_address(match_field),
+            "Classification": self.classification(match_field),
+            "actor": self.actor(match_field),
+            "GreyNoise_URL": self.url(match_field),
```
Comment (lindsey-w) [inline global_helpers/panther_greynoise_helpers.py:62]: curious why are we mis-casing the keys? (some start with lower some start with cap)

Comment (bseb) [reply global_helpers/panther_greynoise_helpers.py:62]: Oh, not intentional, thank you

File: global_helpers/panther_greynoise_helpers.py (hunk 2 above)
Comment (bseb) [reply global_helpers/panther_greynoise_helpers.py:57]: Right? Thank you,  it came to me while doing something completely unrelated over the weekend.  Brains are weird.

File: global_helpers/panther_greynoise_helpers.py (hunk 3 above)
Comment (bseb) [reply global_helpers/panther_greynoise_helpers.py]: Fixed

**Review by lindsey-w — APPROVED** [review]

**Review by rleighton — APPROVED** [review]
LGTM. Please test this change in a test env before we release to customers.

**Comment by bseb** [comment]
Testing looked good, Changes were applied when  uploaded. I did find one method that had the incorrect path to a key, that has been corrected

**Review by nhakmiller — APPROVED** [review]
LGTM

**Pull request merged**